cpp_tests/feature/one_hot_encoder_test.cc

```diff
@@ -14,79 +14,104 @@
  * limitations under the License.
  */
 
-#include "fstream"
-#include <cstdlib>
-#include <string>
-#include <sys/stat.h>
-
 #include "clink/cpp_tests/test_util.h"
 #include "clink/feature/one_hot_encoder.h"
-#include "nlohmann/json.hpp"
 #include "gtest/gtest.h"
 
-namespace tfrt {
+namespace clink {
+
 namespace {
 
 TEST(OneHotEncoderTest, Param) {
-  clink::OneHotEncoderModel model;
-  model.setDropLast(false);
-  EXPECT_EQ(model.getDropLast(), false);
-  model.setDropLast(true);
-  EXPECT_EQ(model.getDropLast(), true);
+  RCReference<OneHotEncoderModel> model =
+      tfrt::TakeRef(test::test_host_context->Construct<OneHotEncoderModel>(
```

Comment:
Instead of using `test::test_host_context` (which is a global variable defined in test_util.h), could we have a before/after method (similar to Junit tests) to initialize e.g. `host_context_ = CreateHostContext(...)` as a class private variable? It is in general not recommended to use global variables in tests.

Reply:
OK. I'll put them in tests classes.